pkg/platform/runtime/setup/setup.go

```diff
@@ -282,7 +282,11 @@ func aggregateErrors() (chan<- error, <-chan error) {
 
 func (s *Setup) installArtifacts(buildResult *model.BuildResult, artifacts artifact.ArtifactRecipeMap, alreadyInstalled store.StoredArtifactMap, setup Setuper) error {
 	if !buildResult.BuildReady && buildResult.BuildEngine == model.Camel {
-		return locale.NewInputError("build_status_in_progress", "", apimodel.ProjectURL(s.target.Owner(), s.target.Name(), s.target.CommitUUID().String()))
+		messageURL := apimodel.ProjectURL(s.target.Owner(), s.target.Name(), s.target.CommitUUID().String())
+		if s.target.Owner() == "" && s.target.Name() == "" {
+			messageURL = apimodel.CommitURL(s.target.CommitUUID().String())
```

Comment:
I don't think this URL shows build progress. Let's bring this up with Akash.

Reply:
We don't need to fix this in this story but we need a follow-up to get the UX before release.
Placeholder PT: https://www.pivotaltracker.com/story/show/178590174

We can leave it as-is for now